modules/kafka/kafka_test.go

```diff
@@ -66,6 +66,32 @@ func TestKafka(t *testing.T) {
 	require.Truef(t, strings.EqualFold(string(consumer.message.Value), "value"), "expected value to be %s, got %s", "value", string(consumer.message.Value))
 }
 
+func TestKafka(t *testing.T) {
+	testCases := []struct {
+		name  string
+		image string
+	}{
+		{
+			name:  "confluentinc",
+			image: "confluentinc/confluent-local:7.5.0",
```

Comment:
Minimal supported version is 7.4.0 so it is reasonable to add a test for 7.4.0 too.